ansible/roles/webserver-content/tasks/main.yml

```diff
@@ -27,12 +27,26 @@
     - "{{ connectbox_default_content_root }}"
     - "{{ connectbox_captive_portal_root }}"
 
-# Copy client interface files
-- name: Copy connectbox client interface
+# Copy local client interface files
+- name: Copy local connectbox client interface
+  when: connectbox_client_repo == ""
```

Comment:
It’s not clear why these `when:` statements exist. I can imagine specifying an alternate repo URL would help with development, but it’s unclear why the repo would ever be undefined. If they need to be here, please add a brief comment on why the `when:` statements exist.